scripts/exaqm_make_ics.sh

```diff
@@ -13,583 +13,13 @@ export pgm=aqm_make_ics
 #-----------------------------------------------------------------------
 #
 . $USHaqm/source_util_funcs.sh
- source_config_for_task "task_make_ics|task_get_extrn_ics" ${GLOBAL_VAR_DEFNS_FP}
-#
-#-----------------------------------------------------------------------
-#
-# Save current shell options (in a global array).  Then set new options
-# for this script/function.
-#
-#-----------------------------------------------------------------------
-#
-{ save_shell_opts; . $USHaqm/preamble.sh; } > /dev/null 2>&1
-#
-#-----------------------------------------------------------------------
-#
-# Get the full path to the file in which this script/function is located
-# (scrfunc_fp), the name of that file (scrfunc_fn), and the directory in
-# which the file is located (scrfunc_dir).
-#
-#-----------------------------------------------------------------------
-#
-scrfunc_fp=$( $READLINK -f "${BASH_SOURCE[0]}" )
-scrfunc_fn=$( basename "${scrfunc_fp}" )
-scrfunc_dir=$( dirname "${scrfunc_fp}" )
-#
-#-----------------------------------------------------------------------
-#
-# Print message indicating entry into script.
-#
-#-----------------------------------------------------------------------
-#
-print_info_msg "
-========================================================================
-Entering script:  \"${scrfunc_fn}\"
-In directory:     \"${scrfunc_dir}\"
-
-This is the ex-script for the task that generates initial condition
-(IC), surface, and zeroth hour lateral boundary condition (LBC0) files
-(in NetCDF format) for the FV3-LAM.
-========================================================================"
-#
-#-----------------------------------------------------------------------
-#
-# Set OpenMP variables.
-#
-#-----------------------------------------------------------------------
-#
-export KMP_AFFINITY=${KMP_AFFINITY_MAKE_ICS}
-export OMP_NUM_THREADS=${OMP_NUM_THREADS_MAKE_ICS}
-export OMP_STACKSIZE=${OMP_STACKSIZE_MAKE_ICS}
```

Comment:
Run time environment is handled by the UW Driver, and isolated from any workflow environment present in the run-script.

That means there will be no collisions between Python workflow environment and component run-time environment.